src/main/java/ch/uzh/ifi/hase/soprafs23/rest/dto/UserGetDTO.java

```diff
@@ -1,13 +1,20 @@
 package ch.uzh.ifi.hase.soprafs23.rest.dto;
 
 import ch.uzh.ifi.hase.soprafs23.constant.UserStatus;
+import java.util.HashSet;
+import java.util.Collections;
+import java.util.Arrays;
+import java.util.Set;
+
+
+
 
 public class UserGetDTO {
 
   private Long id;
   private String username;
   private UserStatus status;
-  private String allergies;
+  private Set<String> allergiesSet;
```

Comment:
I'm unsure about the renaming... why not leave it allergies, as in the entity?

Reply:
I like that its highlighting being a set. So we have allergy and AllergiesSet. Helps me understanding the code, especially while having this here: 
![grafik](https://user-images.githubusercontent.com/91547040/234508499-5d758bd2-11b5-4489-bb27-94d359503b41.png)